pages/register.js

```diff
@@ -0,0 +1,33 @@
+const { I } = inject();
+
+module.exports = {
+firstNameField: {xpath: '//*[@id="input-firstname"]'},
+lastNameField: {xpath: '//*[@id="input-lastname"]'},
+emailField: {xpath: '//*[@id="input-email"]'},
+phoneNumberField: {xpath: '//*[@id="input-telephone"]'},
+passwordField: {xpath: '//*[@id="input-password"]'},
+passwordRepeatField: {xpath: '//*[@id="input-confirm"]'},
+subscribeRatioButton: {xpath: '//*[@id="content"]/form/fieldset[3]/div/div/label[2]'},
+iHaveReadRatioButton: {xpath: '//*[@id="content"]/form/div/div/input[1]'},
+continueButton: {xpath: '//*[@id="content"]/form/div/div/input[2]'},
+
+    verifyRegisterPage() {
+      I.see('Регистрация');
+    },
+
+    fillNewUserForm(user) {
```

Comment:
Rename function, since you not only fill form, but also submit. E.g. submitNewUserForm()